### DISCUSSION TITLE
links to share in dashboards titles

### DISCUSSION THREAD

**Discussion by carza06** [apache/superset#30399] [Q&A / Help]
## Screenshot

![Captura desde 2024-09-25 17-44-59](https://github.com/user-attachments/assets/16bd2205-fe94-429c-8082-3d4ee0b69d93)

## Description

Is it possible to remove the sharing links from the dashboards?

**Comment by rusackas** [comment]
It looks like there's a userCanShare permission, you can try flipping that (though, I'm not an expert on that particular perm).

Converting to a discussion, since this isn't really a bug :)

**Reply by carza06** [reply]
I don't see that permission in the permissions module, and i need hide that link for security of my system
